Notification Service/NotificationService.swift

```diff
@@ -61,7 +74,8 @@ class NotificationService: UNNotificationServiceExtension {
             if Parse.currentConfiguration.isNil  {
                 let config = ParseClientConfiguration { configuration in
                     configuration.applicationGroupIdentifier = Config.shared.environment.groupId
-                    configuration.containingApplicationBundleIdentifier = "com.Jibber-iOS"
+                    // Allow parse to access the data from the main app bundle.
+                    configuration.containingApplicationBundleIdentifier = "com.Jibber-Inc.iOS"
```

Comment:
We had the wrong bundle id which prevented us from getting the token from Parse.